docs/hub/spaces-overview.md

```diff
@@ -66,7 +66,7 @@ If your app requires environment variables (for instance, secret keys or tokens)
 </div>
 
 
-Variables are publicly accessible and viewable and will be automatically added to Spaces duplicated from your repository. They are exposed to your app as environment variables. For Docker Spaces, check out [environment management with Docker](./spaces-sdks-docker#secrets-and-variables-management).
+Variables are publicly accessible and viewable and will be automatically added to Spaces duplicated from your repository. They are exposed to your app as environment variables. For Static Spaces, they are available through client-side JavaScript in `window.huggingface.variables`. For Docker Spaces, check out [environment management with Docker](./spaces-sdks-docker#secrets-and-variables-management).
```

Comment:
(nit) jumping lines to spearate more Docker vs Static

A link to `spaces-sdk-static` could be nice too, wdyt?

```suggestion
Variables are publicly accessible and viewable and will be automatically added to Spaces duplicated from your repository. They are exposed to your app as environment variables.

For Static Spaces, they are available through client-side JavaScript in `window.huggingface.variables`.

For Docker Spaces, check out [environment management with Docker](./spaces-sdks-docker#secrets-and-variables-management).
```